browser/ui/webui/brave_tip_ui.cc

```diff
@@ -278,6 +278,28 @@ void RewardsTipDOMHandler::OnPublisherBanner(
       "brave_rewards_tip.publisherBanner", result);
 }
 
+std::string ConstructTweetIntentURL(const std::string& name,
```

Comment:
we should move this business logic into ledger code so all platforms can use it. So brave_tip_ui should call ledger to get this url and UI should be only responsible for opening it